.github/workflows/e2e-tests-ondemand.yaml

```diff
@@ -6,16 +6,49 @@ on:
       target:
         description: 'Target branch to run E2E tests over'
         required: true
-        # TODO: @andriisoldatenko revisit it
         default: 'release-1.1'
```

Comment:
I guess with every new release we have to take care of bumping this (maybe we can automate it)

Reply:
it was not in AC ;) Feel free to create new ticket

Reply:
:+1: Just brainstorming for future iterations